backend/formatters.in

```diff
@@ -0,0 +1,7 @@
+# Development tools:
+black
+isort
+flake8
+pre-commit
+pyupgrade
+gitlint
```

Comment:
I have a conflict of versions. 
```
ERROR: Cannot install gitlint-core[trusted-deps]==0.19.1 and sh==2.0.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested sh==2.0.3
    gitlint-core[trusted-deps] 0.19.1 depends on sh>=1.13.0; sys_platform != "win32"
    gitlint-core[trusted-deps] 0.19.1 depends on sh==1.14.3; sys_platform != "win32" and extra == "trusted-deps"
``` 
Please check if you can replicate it and, if yes, fix that.

Reply:
After fixing problems with missing package in tests it seems to have resolved itself

Reply:
This conflict is still happening on my machine. I replicated it on a clean Docker container running Python 3.10 image (`python:3.10`). 
```
ERROR: Cannot install gitlint-core[trusted-deps]==0.19.1 and sh==2.0.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested sh==2.0.3
    gitlint-core[trusted-deps] 0.19.1 depends on sh>=1.13.0; sys_platform != "win32"
    gitlint-core[trusted-deps] 0.19.1 depends on sh==1.14.3; sys_platform != "win32" and extra == "trusted-deps"

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict


```

Reply:
FIxed it. Now there is no error when I run script on clean Docker